source/gameengine/Ketsji/KX_GameObject.cpp

```diff
@@ -3530,6 +3530,14 @@ PyObject *KX_GameObject::PySetParent(PyObject *args)
 	}
 	if (!ConvertPythonToGameObject(logicmgr, pyobj, &obj, true, "gameOb.setParent(obj): KX_GameObject"))
 		return NULL;
+
+	if (obj->GetScene()->GetInactiveList()->SearchValue((CValue *)obj) ||
```

Comment:
I think we can allow to parenting an inactive object to another inactive object. In this case the condition will looks like:
```
if (scene->GetInactiveList()->SearchValue(obj) != scene->GetInactiveList()->SearchValue(this))
```
Which will be false if the parent is inactive and the child active or if the parent is active and the child inactive.
Commonly if they are not in the same state.

Also the condition should be moved in `KX_GameObject::SetParent`, because this function is called from python and from the actuator too.